server/app/middleware/question.js

```diff
@@ -0,0 +1,87 @@
+import {
+  getAll,
+  getOne,
+  postQuestion,
+  postAnswer,
+  deleteQuestion,
+  favAnswer,
+} from './../controller/question';
+/** Get all question
+ * @param {obj}
+ * @return {obj}
+ * @public
+*/
+const allquestion = (req, res) => {
+  getAll().then((questions) => {
+    res.send({
```

Comment:
Nice. You should structure your response to return an object with status, message and data(when applicable). It describes the nature of the response better.